lib/src/components/NxTextInput/NxTextInput.scss

```diff
@@ -33,6 +34,7 @@
 }
 
 .nx-text-input__invalid-message {
+  @include regular();
```

Comment:
This is only necessary because the `bold` on the label is being applied to the whole `nx-label` not just `nx-label__text` which is all we actually want it to affect.

Reply:
> This is only necessary because the `bold` on the label is being applied to the whole `nx-label` not just `nx-label__text` which is all we actually want it to affect.

https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/d79d81ea38f5afcf06cb6bc37f753e43d8f8372d